notebooks/examples/configure_check_conditions.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<p>In order to add a condition to an existing check, we can use any of the pre-defined conditions for that check. The naming convention for the methods that add the condition is <code>add_condition_...</code></p><p>If you want to create and add your custom condition logic for parsing the check's result value, see Add a Custom Condition [Link]</p><p> </p><p>--</p><p>New cell (for the sub-subtitle)</p><h3>Add a condition to a new check</h3>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>